document/core/exec/numerics.rst

```diff
@@ -1620,3 +1882,70 @@ Conversions
    \begin{array}{lll@{\qquad}l}
    \reinterpret_{t_1,t_2}(c) &=& \bits_{t_2}^{-1}(\bits_{t_1}(c)) \\
    \end{array}
+
+
+.. _op-sat_u:
+
+:math:`\satu_N(i)`
+.......................
+
+* If :math:`i` is greater than :math:`2^N-1`, return :math:`2^N-1`.
+
+* Else return :math:`i`.
+
+.. math::
+   \begin{array}{lll@{\qquad}l}
+   \satu_N(i) &=& 2^N-1 & (\iff i > 2^N-1)\\
+   \satu_N(i) &=& i & (\otherwise)
+   \end{array}
+
+
+.. _op-sat_s:
+
+:math:`\sats_N(i)`
+.......................
+
+* Let :math:`j` be the :ref:`signed interpretation <aux-signed>` of :math:`i` of size :math:`N`.
```

Comment:
This function is not always called with signed integers:
- `\isubsatu_N(i_1, i_2)` calls it with the result of `i_1 - i_2`.
- `\addsatu_N(i_1, i_2)` calls it with the result of `i_1 + i_2`.

I can change these two calls so that we can drop the reinterpretation step here.

Reply:
Oh oops, this is sat_s, i was looking at sat_u, ignore my comment.